mailbox/event/json/src/main/scala/org/apache/james/event/json/MailboxEventSerializer.scala

```diff
@@ -164,7 +165,8 @@ private object ScalaConverter {
     user = event.getUsername,
     path = MailboxPath.fromJava(event.getMailboxPath),
     mailboxId = event.getMailboxId,
-    added = event.getAdded.asScala.view.mapValues(DTOs.MessageMetaData.fromJava).toMap)
+    added = event.getAdded.asScala.view.mapValues(DTOs.MessageMetaData.fromJava).toMap,
+    isDelivery = IsDelivery(event.isDelivery))
```

Comment:
Please handle format migration: do we support when this field is missing? Akka legacy format?

(write a unit test for this...)